README.md

```diff
@@ -224,13 +224,23 @@ or using shadow builds.
 
 ## Building for Sailfish SDK
 
-1. create set of directories like '~/QtCreator/Build' '~/QtCreator/Install'
-2. navigate to '~/QtCreator' and clone the Qt Creator's source tree there
-3. run qmake:
-	qmake ../digia-qt-creator/qtcreator.pro -r -after "DEFINES+=IDE_COPY_SETTINGS_FROM_VARIANT=. IDE_SETTINGSVARIANT=SailfishBeta6" QTC_PREFIX=
+Building Qt Creator for the Sailfish SDK is done using some helper scripts,
+which are available in a github repo:
 
-4. build QtCreator, intaller, artifacts etc as defined in
-	http://qt-project.org/wiki/Building-Qt-Creator-Packages
+    https://github.com/sailfishos/sdk-build-tools.git
+
+Before building Qt Cretor, you must build the prerequisite libraries (ICU, Qt
```

Comment:
```suggestion
Before building Qt Creator, you must build the prerequisite libraries (ICU, Qt
```

Reply:
This is weird, I noticed that typo when reading through what I had written, and I clearly remember fixing it :-O